### PR TITLE
Added requirements for connectivity and sms

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/users-guide/administration-bundle/changing-settings.md [hunk 1]
```diff
@@ -263,6 +265,13 @@ For further information, see [http://enable-cors.org](http://enable-cors.org).
 <a name="properties"></a>
 ### Managing the properties library
 
+{{< c8y-admon-req >}}
+ROLES & PERMISSIONS:
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/changing-settings.md:269]: I only found this sentence and moved it into the requirements section. I didn´t find any info which team is responsible for the properties library.

File: content/users-guide/administration-bundle/changing-settings.md [hunk 2]
```diff
@@ -352,6 +357,14 @@ Custom properties are visible to all authenticated users of the tenant, regardle
 <a name="sms-provider"></a>
 ### Providing SMS provider credentials
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/changing-settings.md:358]: @l2p- Please have this reviewed and if needed enhanced by your team.

File: content/users-guide/administration-bundle/changing-settings.md [hunk 3]
```diff
@@ -378,7 +387,18 @@ OpenIT does not serve new customers anymore and is in the process of shutting do
 <a name="connectivity"></a>
 ### Managing the connectivity settings
 
-In the **Connectivity** page, you can manage credentials for different providers. In order to add or replace credentials ADMIN permissions are required.
+{{< c8y-admon-req >}}
+ROLES & PERMISSIONS:
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/changing-settings.md:391]: @l2p- Please have this reviewed and if needed enhanced by your team.

**Comment by BeateRixen** [comment]
I have added some more requirements for the "Changing settings" chapter. I only added what I could find in the text, i.e. the requirements for connectivity and SMS. For applications and properties library, I assume the PlaMa team is responsible? Couldn´t find it in the feature & teams sheet.

File: content/users-guide/administration-bundle/changing-settings.md [hunk 4]
```diff
@@ -352,6 +357,14 @@ Custom properties are visible to all authenticated users of the tenant, regardle
 <a name="sms-provider"></a>
 ### Providing SMS provider credentials
 
+{{< c8y-admon-req >}}
+ROLES & PERMISSIONS:
+
+To view SMS provider configurations: READ permission for the permission type "SMS"
+To modify SMS provider configurations: ADMIN permission for the permission type "SMS"
```
Comment (l2p-) [inline content/users-guide/administration-bundle/changing-settings.md:364]:
```suggestion
To set or remove SMS provider configurations: ADMIN permission for the permission type "SMS"
```

File: content/users-guide/administration-bundle/changing-settings.md [hunk 5]
```diff
@@ -378,7 +387,18 @@ OpenIT does not serve new customers anymore and is in the process of shutting do
...
 ### Managing the connectivity settings
 
-In the **Connectivity** page, you can manage credentials for different providers. In order to add or replace credentials ADMIN permissions are required.
+{{< c8y-admon-req >}}
+ROLES & PERMISSIONS:
+
+The **Connectivity** menu item is only available if you are logged in to the {{< product-c8y-iot >}} platform as administrator and if you have READ or ADMIN permission for the permission type "Connectivity".
+
+To view connectivity settings: READ permission for the permission type "Connectivity"
+To add or replace credentials: ADMIN permission for the permission type "Connectivity"
```
Comment (l2p-) [inline content/users-guide/administration-bundle/changing-settings.md:396]:
```suggestion
To set or remove connectivity provider configuration: ADMIN permission for the permission type "Connectivity"
```

**Review by l2p- — CHANGES_REQUESTED** [review]

File: content/users-guide/administration-bundle/changing-settings.md (hunk 1 above)
Comment (Airdroplet) [reply content/users-guide/administration-bundle/changing-settings.md:269]: This looks ok from my perspective. Properties lib belong to Platform team.

**Review by Airdroplet — APPROVED** [review]

File: content/users-guide/administration-bundle/changing-settings.md [hunk 6]
```diff
@@ -378,7 +387,18 @@ OpenIT does not serve new customers anymore and is in the process of shutting do
...
 ### Managing the connectivity settings
 
-In the **Connectivity** page, you can manage credentials for different providers. In order to add or replace credentials ADMIN permissions are required.
+{{< c8y-admon-req >}}
+ROLES & PERMISSIONS:
+
+The **Connectivity** menu item is only available if you are logged in to the {{< product-c8y-iot >}} platform as administrator and if you have READ or ADMIN permission for the permission type "Connectivity".
+
+To view connectivity settings: READ permission for the permission type "Connectivity"
+To set or remove connectivity provider configuration: ADMIN permission for the permission type "Connectivity"
```
Comment (rssag) [inline content/users-guide/administration-bundle/changing-settings.md:396]:
```suggestion
To set or remove connectivity provider configurations: ADMIN permission for the permission type "Connectivity"
```

**Review by rssag — APPROVED** [review]
1 suggestion + approval

**Pull request merged**